assembly/pallets/balances/assembly/balances-module.ts

```diff
@@ -0,0 +1,99 @@
+import { u128 } from "as-bignum";
+import { ByteArray, BytesReader, UInt64 } from "as-scale-codec";
+import { AccountData, ISignedTransaction, Log, ResponseCodes, Storage, TransactionValidity } from 'subsembly-core';
+import { System } from "../../../frame";
+import { AccountIdType, Balance } from "../../../runtime/runtime";
+
+/**
+ * @description The Balances Module.
+ * Used for account balance manipulation such as:
+ *  - Getting and setting free/reserved balances
+ */
+export class Balances {
+
+    /**
+     * @description Returns AccountData for a given AccountId
+     * If the account does not exist, Default AccountData is returned.
+     */
+    static getAccountData(accountId: AccountIdType): AccountData {
+        const accDataBytes = Storage.get(accountId.getAddress());
+        if (accDataBytes.isSome()) {
+            return BytesReader.decodeInto<AccountData>((<ByteArray>accDataBytes.unwrap()).values);
+        } else {
+            return AccountData.getDefault();
+        }
+    }
+
+    /**
+     * @description Sets the balances of a given AccountId
+     * Alters the Free balance and Reserved balances in Storage.
+     */
+    static setBalance(accountId: AccountIdType, freeBalance: Balance, reservedBalance: Balance): AccountData {
+        const currentAccountData = this.getAccountData(accountId);
+
+        // TODO Any meaningful checks
+
+        currentAccountData.setFree(freeBalance);
```

Comment:
I think that our Balances module uses `U128` and if type `Balance` is different than u128 it won't compile.
Maybe we should use `IAccountData` and not `AccountData` or if it won't work that way we might need to move `accountdata` from `core`?